manifests/config/apache.pp

```diff
@@ -184,22 +184,8 @@
   }
 
   # mod_proxy supports "ProxyPass ... upgrade=websocket" since 2.4.47
-  # EL8: 2.4.37 / EL9: 2.4.62 / Debian11: 2.4.62 / Ubuntu20.04: 2.4.41 / Ubuntu22.04: 2.4.52
-  $proxy_upgrade_websocket = !($facts['os']['family'] == 'RedHat' and $facts['os']['release']['major'] == '8') and !($facts['os']['name'] == 'Ubuntu' and $facts['os']['release']['major'] == '20.04')
-  if $proxy_upgrade_websocket {
-    $vhost_rewrites = []
-    $_proxy_params = $proxy_params + { 'upgrade' => 'websocket' }
-  } else {
-    include apache::mod::proxy_wstunnel
-    $websockets_backend = regsubst($_proxy_backend, 'http://', 'ws://')
-    $websockets_rewrite = {
-      'comment'      => 'Upgrade Websocket connections',
-      'rewrite_cond' => '%{HTTP:Upgrade} =websocket [NC]',
-      'rewrite_rule' => "/(.*) ${websockets_backend}\$1 [P,L]",
-    }
-    $vhost_rewrites = [$websockets_rewrite]
-    $_proxy_params = $proxy_params
-  }
+  $vhost_rewrites = []
```

Comment:
If this is always empty, I think we can simply drop the definition and its use on line 216.

Reply:
fixed it